### PR TITLE
Updates for ALZ Bicep Accelerator

### PR DESCRIPTION
**Pull request by oZakari**
# Pull Request

## Issue
General adjustments for ALZ Bicep Accelerator

Issue #, if available:

## Description

Description of changes:
- Need to adjust the policy assignment param file as switched to the ALZ defaults.
- Need to switch to orchestration version of management diagnostic setting module param file.
- Switching custom modules directory to match naming convention of custom parameters directory.

## License

By submitting this pull request, I confirm that my contribution is made under the terms of the projects associated license.

### CODE REVIEW

**Comment by Nepomuceno** [comment]
This willl not change to use the prerelease version that does have this new files you would need to change the version to use that too.

**Review by Nepomuceno — CHANGES_REQUESTED** [review]
This will Requiure us to change the version on the files too.

**Review by Nepomuceno — APPROVED** [review]

File: src/ALZ/Assets/alz-bicep-config/v0.14.1-pre.config.json [hunk 1]
```diff
@@ -23,12 +23,12 @@
             "destination": "config/custom-parameters/managementGroups.parameters.all.json"
         },
         {
-            "source": "infra-as-code/bicep/modules/mgDiagSettings/parameters/mgDiagSettings.parameters.all.json",
-            "destination": "config/custom-parameters/mgDiagSettings.parameters.all.json"
+            "source": "infra-as-code/bicep/orchestration/mgDiagSettingsAll/parameters/mgDiagSettingsAll.parameters.min.json",
```
Comment (lovelysandwich) [inline src/ALZ/Assets/alz-bicep-config/v0.14.1-pre.config.json:26]: There are a few values in here we could do with creating Computed substitutions and replacements for.   I can help with this if you'd like.

File: src/ALZ/Assets/alz-bicep-config/v0.14.1-pre.config.json [hunk 2]
```diff
@@ -23,12 +23,12 @@
...
+            "destination": "config/custom-parameters/mgDiagSettingsAll.parameters.all.json"
         },
         {
-            "source": "infra-as-code/bicep/modules/policy/assignments/parameters/policyAssignmentManagementGroup.dine.parameters.all.json",
-            "destination": "config/custom-parameters/policyAssignmentManagementGroup.dine.parameters.all.json"
+            "source": "infra-as-code/bicep/modules/policy/assignments/alzDefaults/parameters/alzDefaultPolicyAssignments.parameters.all.json",
```
Comment (lovelysandwich) [inline src/ALZ/Assets/alz-bicep-config/v0.14.1-pre.config.json:30]: alzDefaultPolicyAssignments.parameters.all.json has quite a few xxxx-ed out values.  The log analytics workspace id is probably a straightforward one - but the Ddos and Dns based Ids might need some thought...  Presumably these are values from the connectivity subscription.

File: src/ALZ/Private/New-ALZDirectoryEnvironment.ps1 [hunk 3]
```diff
@@ -10,7 +10,7 @@ function New-ALZDirectoryEnvironment {
     # Create destination file structure
     $gitHubPipeline = Join-Path $alzEnvironmentDestination ".github" "workflows"
     $config = Join-Path $alzEnvironmentDestination "config"
-    $configModules = Join-Path $alzEnvironmentDestination "config" "modules"
+    $configModules = Join-Path $alzEnvironmentDestination "config" "custom-modules"
```
Comment (lovelysandwich) [inline src/ALZ/Private/New-ALZDirectoryEnvironment.ps1:13]: See how quickly I lose context after a few days away :D  This is adopting a directory name change in ALZ-Bicep?

**Review by lovelysandwich — APPROVED** [review]
Really nice work!

**Pull request merged**